src/content/pages/program.md

```diff
@@ -1,9 +1,272 @@
+- Who are new to Python,
+- Who have built interesting things in Python,
+- Who have broken interesting things with Python,
+- Who have helped build Python, and
+- Who have expertise to share with our community from fields outside of Python (and outside of technology).
+
+Our CFP process also encompasses submissions to the Specialist Tracks (Education, Scientific Python, and DevOops), so you can submit your talk to the main conference and the Specialist Tracks with one click.
+
+To submit your proposal to one or more of the Specialist Tracks, please tag it with the track names of your choice, which are provided for you in the screens that follow. You can select more than one!
+
+**If you have questions about the CFP process, you can reach us at [program@pycon.org.au](mailto:program@pycon.org.au).**
+
+## Specialist Tracks
+
+This year we have three specialist tracks you can submit to, in addition to the main conference! Specialist tracks are independently run events held on the first day of PyCon AU: Scientific Python, Education, and DevOops.
+
+## _NEW: Academic Abstract Submission_
+
+Are you an academic researcher or otherwise very excited about the prospect of publishing your work as a peer-reviewed paper? If so, read on. (If not, you can ignore this whole section and learn more about [talk slots and timing](#talk-slots-and-timing), below.)
+
+This year, PyCon AU is partnering with the [Journal of Open Source Software (JOSS)](https://joss.theoj.org/) to publish academic papers in association with PyCon AU submissions.
+
+_Academic abstract submission will not increase your chances of having your talk selected for presentation at PyCon AU._ This is intended as a way for participants to further engage with the conference and build their academic track record. If your proposal is accepted as part of the PyCon AU program, we will work with you to handle the JOSS submission, and upon acceptance it will become part of a new PyCon AU collection.
+
+Your academic abstract submission must fulfil basic requirements for publication in JOSS:
+* Your submission must describe open source software ([as defined by OSI](https://opensource.org/osd))
+* Your software must have an obvious research application
+* You must be a major contributor to the software
+
+Additionally, any submissions must comply with the PyCon AU [Code of Conduct](/conduct), as with their accompanying presentation.
+
+For this academic submission, you must provide:
+* A title
+* Authors names and affiliations of all authors
+* Academic abstract text itself, up to 500 words.
+
+You do not have to worry about adding references at this stage (although a few key references might help us evaluate the work - we may not have a reviewer on hand who works in your field).
+
+If your proposal moves forward, members of the PyCon AU academic program committee will work with you to develop your academic abstract submission into a JOSS paper that complies with the journal standards.
```

Comment:
Would it be appropriate to link to the JOSS docs https://joss.readthedocs.io/ ? 
Probably unnecessary now, can add later but this sentence opened this tab in my mind.